### ISSUE TITLE
Add Plangothic

### ISSUE THREAD

**Issue by hfhchan** [google/fonts#4746]
**Font Project Git Repo URL:**
https://github.com/Fitzgerald-Porthmouth-Koenigsegg/Plangothic

**Super short description of the Font Family:**
An project based on extending Source Han Sans targeting to cover all of Unicode.

It currently covers all of CJK Unified Ideographs Extension G!

**Requirements:**

I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to namecheck.fontdata.com
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [ ] The repository respect the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure)
- [ ] The source files are available in the repository
- [ ] I read, agree and comply with the full [contributing requirements](https://github.com/google/fonts/blob/main/CONTRIBUTING.md)
- [ ] I am the author of the font or the author agreed to this submission
- [ ] I will maintain the repository

**Image:**

<img src="https://raw.githubusercontent.com/Fitzgerald-Porthmouth-Koenigsegg/Plangothic/main/pic/31.png">

**Notes:**
I do not own this project, and I am unsure if the original copyright holders have signed the Google CLA. However this will be good project for Google Fonts to check up with, because it will be more "Noto" than the existing "Noto Sans SC".

**Comment by Fitzgerald-Porthmouth-Koenigsegg** [comment]
Thanks for acknowledging my work

**Comment by davelab6** [comment]
@punchcutter @marekjez86 @simoncozens do you have any suggestions how this can be integrated upstream?

**Comment by Fitzgerald-Porthmouth-Koenigsegg** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Fitzgerald-Porthmouth-Koenigsegg** [comment]
I agree with @hfhchan's request for fonts, but there are a few questions:
- The font is still in the production stage, CJK Unified Ideographs Extension G can be proposed and submitted separately, but the entire Plangothic cannot be submitted directly because it has not been screened yet.
- A considerable number of people participated in the production of this font, so I can't accurately judge all the people who participated in the project (although currently @Hulenkius and I are in the dominant position).

**Comment by Fitzgerald-Porthmouth-Koenigsegg** [comment]
> I agree with @hfhchan's request for fonts, but there are a few questions:
> 
> * The font is still in the production stage, CJK Unified Ideographs Extension G can be proposed and submitted separately, but the entire Plangothic cannot be submitted directly because it has not been screened yet.
> * A considerable number of people participated in the production of this font, so I can't accurately judge all the people who participated in the project (although currently @Hulenkius and I are in the dominant position).

ps: Hentaigana can also be submitted. It even has multiple weights. However, this project seems to have not been updated for a long time. See: https://github.com/Hulenkius/hentaigana-sans

**Comment by Fitzgerald-Porthmouth-Koenigsegg** [comment]
Dear all, greetings. I am the founder of the Plangothic project. 

First of all, the Plangothic project has made significant progress. Apart from the CJK Extension B, all the CJK characters in other blocks have been completed, including the Extension I introduced in Unicode 15.1 and sporadic characters present in WS2021. 

Secondly, I have just signed the Google CLA. 

Lastly, the copyright of this font belongs to the "Plangothic Project Team," which consists of 10 individuals who have all participated in the production of the Plangothic project. 

Due to numerous busy real-life affairs and my unfamiliarity with this matter, I failed to inform about the project's interim achievements in a timely manner. I apologize for my negligence and oversight. As a result, the project has not been completed until now. The remaining unfinished parts will be substituted with the Hanazono Mincho font, which is also an open-source font.

The project will also be continuously maintained, and I will promptly address any flaws that arise in the project.

So I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode)
* [x]  The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
* [x]  There is no "Reserved Font Name" mention in the OFL licenseOFL
* [x]  Family name is unique according to namecheck.fontdata.com
* [x]  The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
* [x]  The repository respect the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure)
* [x]  The source files are available in the repository
* [x]  I read, agree and comply with the full [contributing requirements](https://github.com/google/fonts/blob/main/CONTRIBUTING.md)
* [x]  I am the author of the font or the author agreed to this submission
* [x]  I will maintain the repository

**Image: 图片：**

![](https://raw.githubusercontent.com/Fitzgerald-Porthmouth-Koenigsegg/Plangothic/main/pic/31.png)

**Comment by FlaviusChromacitrin** [comment]
i think its now complete
